### ISSUE TITLE
Dependency errors when trying to use gpt2 using pytorch hub.

### ISSUE THREAD

**Issue by VictorAlbertos** [huggingface/pytorch-transformers#1165]
It started today, yesterday it was working fine. When I try to download `gpt2` model from pytorch hub repository, as follows: 

```python
torch.hub.load('huggingface/pytorch-pretrained-BERT', 'gpt2Tokenizer', 'gpt2') 
```

I get the following error: `ModuleNotFoundError: No module named 'sacremoses'`. If I add that dependency manually then I get another error: `ModuleNotFoundError: No module named 'sentencepiece'`. Then I add `sentencepiece` dependency manually just to get another error: `RuntimeError: Cannot find callable gpt2Tokenizer in hubconf`. And this error seems to be related to API changes. 

I'm using a Google Colab GPU instance.

If this is not the right place to post this issue, please re-direct me to the proper source to post the issue. Thanks.

**Comment by VictorAlbertos** [comment]
I found that https://github.com/huggingface/pytorch-transformers/commit/256086bc6908448fc6aff9b1e19d95c4f6019bee is the source of the issue. Reading the changes I could guess that the new way for retrieving the tokenizer and model is as follows:

```python
tokenizer = torch.hub.load('huggingface/pytorch-transformers', 'tokenizer', 'gpt2')
model = torch.hub.load('huggingface/pytorch-transformers', 'modelWithLMHead', 'gpt2')
```

But I'm not sure if there is an issue with the docs in hub as they seem to not be updated.

**Comment by thomwolf** [comment]
Yes, we are in the process of updating the hub

**Comment by VictorAlbertos** [comment]
I'm reopening this issue because I'm getting the next error when trying to import the tokenizer:
`ImportError: cannot import name 'add_start_docstrings'`

<img width="1046" alt="Screenshot 2019-09-06 at 20 15 36" src="https://user-images.githubusercontent.com/2614726/64450882-4da01080-d0e3-11e9-94d0-10a0e57c0e80.png">

**Comment by thomwolf** [comment]
We can only help you if we have more information on the version/release you
are using.

**Comment by VictorAlbertos** [comment]
I'm using the version from the hub, you don't specify a version there, as far as I know.

**Comment by thomwolf** [comment]
It's the version of the master branch by default.

I fixed the bug with ee027c8.

Note that you can [specify a specific release](https://pytorch.org/docs/stable/hub.html#torch.hub.load) with torch hub, e.g. use release `1.3.0` with `torch.hub('huggingface/pytorch-transformers:1.3.0', 'model', 'bert-base-uncased')`.

That's what I would advise as it allows you to have clean versioning of your code (you will be sure, in 3 months from now, of the exact version of the model you were using to get your results).

**Comment by VictorAlbertos** [comment]
Thanks for fixing the bug so quickly and for the additional information I was not aware of the versioning feature.
